py3status/modules/pingdom.py

```diff
@@ -13,6 +13,7 @@
     login: pingdom login (default '')
     max_latency: maximal latency before coloring the output (default 500)
     password: pingdom password (default '')
+    request_timeout: pindgom API request timeout (default 10)
```

Comment:
@ultrabug You should be able to hide this (still) cause I'm using gettattr here. No?

Reply:
Until this module is not migrated, I'd rather stay like this legacy style